frontend/src/Pages/SignupPage/SignupPage.jsx

```diff
@@ -0,0 +1,148 @@
+                        <TextField id="password" type='password' label="Enter password" variant="filled" name="password" onChange={handleInput} 
+                        value={userInfo.password} error={!!error.password} helperText={error.password} InputProps={{style:{width: 550} }}/>
+                 </Grid>
+                <Grid>
+                        <InputLabel htmlFor="confirmpassword" style={{fontWeight:"bold"}}>Confirm Password</InputLabel>
+                        <TextField id="confirmpassword" type='password' label="Enter password" variant="filled" name="confirmPassword" onChange={handleInput} 
+                        value={userInfo.confirmPassword} error={!!error.confirmPassword} helperText={error.confirmPassword} InputProps={{style:{width: 550} }}/>
+                </Grid>
+            <Grid item style={{display:"flex", flexDirection:"column",alignItems:"flex-start",justifyContent:"flex-end"}}>
+                <InputLabel style={{marginTop:20}}>
+                    <FormControlLabel control={<Checkbox checked={checked} onChange={isChecked} />}
+                    label={
+                        <Typography>
+                            I have read and accept <span style={{fontWeight:"bold"}}>Terms and Conditions</span>
+                        </Typography>
+                    } />
+                </InputLabel>
+                <Button variant="contained" type="submit" style={{alignItems:"center", color:"black", fontWeight:"bold",
+                backgroundColor:"#1B7D4F", marginLeft:"125px", marginTop:"20px",width:300}}>
+                    Register
+                </Button>
+                <div style={{display:"flex",marginTop:20}}>
+                <Typography variant="body" style={{marginLeft:"125px"}}>
+                    Already have an account?</Typography>
+                    <Button variant="text" style={{color:"black", fontWeight:"bold",marginTop:-5}}>Sign in!</Button>
+                </div>
+                </Grid>
+                </form>
+                </Grid>
+                </Grid>
+                <Grid container style={{display:"flex", flexDirection:"column",alignItems: 'center', justifyContent: 'center'}}>
+                    <Divider sx={{ borderBottomWidth: 3, width:"75%"}}/>
+                    <div style={{marginTop:10}} >
+                    <Typography variant="body" fontWeight="bold" >
+                        Or Sign Up With
+                    </Typography></div>
+                    <Grid container spacing={3} style={{display:"flex", flexDirection:"row",alignItems: 'center', justifyContent: 'center'}} >
+                        <Grid item >
+                            <IconButton color="primary">
+                                <GoogleIcon/></IconButton>
+                        </Grid>
+                    </Grid>
+                </Grid>
+            </Grid>
+        </Grid>
+        </main>
+    );
+}
+
+export default SignupPage;
```

Comment:
Awesome work! Only note is to wrap this line in a memo because the registration form changes will most likely cause a lot of reloads throughout the site that's working in the background. 

export default memo(SignupPage);